src/vm/memory/relocatable.zig

```diff
@@ -224,6 +252,18 @@ pub const MaybeRelocatable = union(enum) {
             .felt => false,
         };
     }
+
+    /// Returns whether the MaybeRelocatable is a felt or not.
+    ///
+    /// # Returns
+    ///
+    /// `true` if the MaybeRelocatable is a felt, `false` otherwise.
+    pub fn isFelt(self: MaybeRelocatable) bool {
+        return switch (self) {
+            .relocatable => false,
+            .felt => true,
+        };
```

Comment:
```suggestion
        return std.meta.activeTag(self) == .felt
```

Reply:
Oh didn't know about `.activeTag`. That's great.

I'll change the `.isRelocatable` function as well.